pyrecdp/encoder.py

```diff
@@ -131,7 +134,7 @@ def __init__(self, proc, x_col_list, y_col_list, out_col_list, out_name):
         if len(self.out_col_list) < self.expected_list_size:
             raise ValueError("CountEncoder __init__, input out_col_list should be same size as y_col_list")
 
-    def transform(self, df):
+    def transform(self, df, train_generate=True):
```

Comment:
@XinyaoWa , I understand your purpose here, but I want to make transform method a general api, so It would be best that the argument doesn't change for different encoder.